node/t3rn-parachain/src/chain_spec.rs

```diff
@@ -178,7 +179,7 @@ fn polkadot_genesis(
             balances: endowed_accounts
                 .iter()
                 .cloned()
-                .map(|k| (k, 1 << 60))
+                .map(|k| (k, 100_000_000_000000000000))
```

Comment:
```suggestion
                .map(|k| (k, 100_000_000_000_000_000_000)) // 100 million TRN
```
doing a complete genesis issuance might lead to undesired sfx such as bogus circulating   supply à la `T::Currency::total_issuance()`. options:
+ partial genesis issuance
+ no issuance until vesting